### PR TITLE
Minor updates 

### PR DESCRIPTION
**Pull request by exactlyallan**
Updated announcements (no more specials), updated blog posts (will need to again for latest release tbd), added networkx to pydata accelerators section.

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -8,45 +8,22 @@
...
+        "text" "Catch up with GTC'24 On-Demand"
+        "url" "https://www.nvidia.com/gtc/"
     )
     (dict
         "text" "RAPIDS 24.04 Released"
         "url" "https://docs.rapids.ai/install#selector"
     )
     (dict
-        "text" "Notice: Deprecation for Pascal GPUs"
+        "text" "Notice: Pascal Deprecation"
```
Comment (bdice) [inline layouts/index.html:26]: Should we change this to a CentOS 7 deprecation, or something else? The Pascal removal went out with RAPIDS 24.02 so we might be able to change this.

**Review by bdice — APPROVED** [review]

File: layouts/index.html (hunk 1 above)
Comment (exactlyallan) [reply layouts/index.html:26]: I was thinking about that too.. I think pascal is a bigger deal than centos?

Comment (bdice) [reply layouts/index.html:26]: I'd also be fine with dropping this notice. cc: @aravenel What do you think?

Comment (beckernick) [reply layouts/index.html:26]: I think we can drop this -- it's no longer critical new information as of 24.04. The requirements section notes Volta+.

**Review by bdice — APPROVED** [review]
Looks good!

**Pull request merged**